E2a-TRAS/SP01/README.md

```diff
@@ -0,0 +1,272 @@
+
+</details>
+
+
+### Object typing
+
+> **Acceptance criteria**: For the **Object typing** capability, the validation procedure must verify that an IFC entity type with the given Name is typing (via `IfcRelDefinesByType`) exactly a given number of objects of the requested Name, no more and no less.
+>
+> If present, all criteria listed in [Usages, Constraints & Logic](#Usages,-Constraints-&-Logic), and in the same section of precondition tests, shall be verified too.
+
+| Entity Type   | Entity Type Name        | Minimum | Maximum | IfcObject | IfcObject Name              |
+|---------------|-------------------------|---------|---------|-----------|-----------------------------|
+| IfcCourseType | Segmento di massicciata | 1       |         | IfcCourse | Segmento di massicciata M01 |
+| IfcRailType   | Rotaia 60E1             | 1       |         | IfcRail   | Rotaia BC01 DX              |
+| IfcRailType   | Rotaia 60E1             | 1       |         | IfcRail   | Rotaia BC01 SX              |
+
+NOTE:
+- when **Minimum** and **Maximum** have the same value, it means exactly. Example: Minimum=Maximum=1, means that the entity type must type exactly 1 object with that Name.
+- when **Maximum is empty**, it means **unlimited**. Example: Minimum=1; Maximum=empty, means that the Entity Type must type 1 or more Object of the requested name.
+
+### Spatial decomposition
+
+> **Acceptance criteria**: For the **Spatial decomposition** capability, the validation procedure must verify that a Parent Element of the requested type aggregates (via `IfcRelAggregates`) exactly a given number of Child Elements of the requested type, no more and no less.
+
+| Parent Element | Parent Element Type | Minimum | Maximum | Child Element   | Child Element Type |
+|----------------|---------------------|---------|---------|-----------------|--------------------|
+| IfcRailway     | Località            | 2       | 2       | IfcFacilityPart | TRACKSTRUCTURE     |
+
+
+### Spatial containment
+
+> **Acceptance criteria**: For the **Spatial containment** capability, the validation procedure must verify that a Spatial Element of the requested type contains (via `IfcRelContainedInSpatialStructure`) exactly a given number of Elements of the requested type, no more and no less.
+
+| Spatial Element | Spatial Element Type | Minimum | Maximum | Element            | Element Type      |
+|-----------------|----------------------|---------|---------|--------------------|-------------------|
+| IfcFacilityPart | TRACKSTRUCTURE       | 1       |         | IfcCourse          | BALLASTBED        |
+| IfcFacilityPart | TRACKSTRUCTURE       | 2       |         | IfcRail            | RAIL              |
+| IfcFacilityPart | TRACKSTRUCTURE       | 1       |         | IfcTrackElement    | SLEEPER           |
+
+NOTE:
+- when **Maximum is empty**, it means **unlimited**. Example: Minimum=1; Maximum=empty, means that the Spatial Element must contain 1 or more elements of the requested type.
+
+
```

Comment:
Could you maybe make a suggested hierachy using bullet point structure to understand the required breakdown?

Reply:
done